fabric-networking-api-v1/src/main/resources/fabric.mod.json

```diff
@@ -40,6 +40,8 @@
   "custom": {
     "fabric-api:module-lifecycle": "stable",
     "loom:injected_interfaces": {
+      "net/minecraft/class_3248": [ "net/fabricmc/fabric/api/networking/v1/ServerTransferable", "net/fabricmc/fabric/api/networking/v1/ServerCookieStore" ],
+      "net/minecraft/class_8609": [ "net/fabricmc/fabric/api/networking/v1/ServerTransferable", "net/fabricmc/fabric/api/networking/v1/ServerCookieStore" ],
```

Comment:
I think we can likely do this without any injected interfaces tbh.

Reply:
Removed